docs/sources/operations/caching.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Caching 
+menuTitle: Caching 
+description: Enable and configure memcached for caching. 
+weight: 100
+keywords:
+  - memcached
+  - caching
+---
+<!-- For more information about how to populate front matter, see [Topic front matter]({{< relref "../../front-matter/" >}}). -->
+
+# Caching
+
+Loki supports caching of index writes and lookups, chunks and query results to
+speed up query performance. This sections describes the recommended Memcached
+configuration to enable caching for chunks and query results. The index lookup
+cache is configured to be in-memory by default.
+
+## Before you begin
+
+- It is recommended to deploy four dedicated Memcached clusters.
+- As of 2023-02-01 the `memcached:1.6.17-alpine` is recommended.
+- Consult the Loki ksonnet [memcached](https://github.com/grafana/loki/blob/main/production/ksonnet/loki/memcached.libsonnet) deployment and the ksonnet [memcached library](https://github.com/grafana/jsonnet-libs/tree/master/memcached).
+
+## Steps
+
+To enable and configure Memcached:
+
+1. Deploy each Memcached service with at least three replicas and configure
+   each as follows:
+    1. Chunk cache `--memory-limit=4096 --max-item-size=2m --conn-limit=1024`
+    1. Query result cache `--memory-limit=1024 --max-item-size=5m --conn-limit=1024`
+
+1. Configure Loki to use the cache.
+    1. If the Helm chart is used
+        1. set `memcached.chunk_cache.host` to the Memecache address of for the chunk cache.
+        1. set `memcached.results_cache.host` to the Memecache address of for the query result cache.
+	1. set `memcached.chunk_cache.enabled=true` and `memcached.results_cache.enabled=true`.
+        1. Ensure that the connection limit of Memcached is at least `number_of_clients? * max_idle_conns`.
+  1. If the Loki configuration is used
```

Comment:
I think something is off with the numbering formatting here, I think this should be nested at the same level as "If the Helm chart is used" under step 2?

Reply:
Yes! Thank you!